packages/coreutils/src/statedb.ts

```diff
@@ -23,21 +23,6 @@ export const IStateDB = new Token<IStateDB>('@jupyterlab/coreutils:IStateDB');
  */
 export interface IStateDB<T extends ReadonlyJSONValue = ReadonlyJSONValue>
   extends IDataConnector<T> {
-  /**
-   * The maximum allowed length of the data after it has been serialized.
-   */
-  readonly maxLength: number;
```

Comment:
Maybe it would be worth warning the users of the `StateDB` that it is frequently sent over the wire, so it is not meant for storing large data.

Reply:
Yeah, that's a good point.